x-pack/test/security_solution_cypress/es_archives/auditbeat_for_exceptions_from_alert/mappings.json

```diff
@@ -0,0 +1,3577 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "exceptions-8.0.0": {
+        "is_write_index": false
+      },
+      "beats": {
+      },
+      "siem-read-alias": {
+      }
+    },
+    "index": "exceptions-8.0.0-2019.08.30-000021",
```

Comment:
I found that this was causing issues because `esArchiver.unload` was having no effect. The data is referencing `exceptionsalerts-` but the index name is `exceptions-`, so esArchiver was not cleaning up the index properly.